install/roles/server/tasks/uninstall_podman_containers.yml

```diff
@@ -34,7 +34,7 @@
   when:
     - have_qpc_server_podman_container
 
-- name: Set variable indicating whether any QPC podman containers found
+- name: Set variable indicating whether any quipucords podman containers found
```

Comment:
change to `Set variable indicating whether any Quipucords podman containers are present`